tag-elements-connector/Api-input/Connector/src/assets/domain_schemas/BisCore.ecschema.xml

```diff
@@ -0,0 +1,2403 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ECSchema schemaName="BisCore" alias="bis" version="01.00.13" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.1" displayLabel="BIS Core" description="The BIS core schema contains classes that all other domain schemas extend.">
```

Comment:
@ashish-srivastava-dev Rather than check in this version of the schema, can't you pull down from npm? Schemas are published there.

e.g. let anschem = require.resolve('@bentley/analytical-schema/Analytical.ecschema.xml');

I also think if you call this.synchronizer.imodel.importSchemas for schemas which reference biscore, you should get that imported for you.  Don't need to import specific version of BisCore.  Is this the only way you can import?  I heard another group report something similar to this and I tried to reproduce, but, I could not.

Reply:
Removed the unnecessary schemas and loading schema from npm package.